record/custom_records.go

```diff
@@ -22,3 +24,8 @@ func (c CustomSet) Validate() error {
 
 	return nil
 }
+
+// IsKeysend checks if the custom records contain the key send type.
+func (c CustomSet) IsKeysend() bool {
```

Comment:
nit: though trivial, we could cover this function with a unit test to not lower coverage.

Reply:
+1

Reply:
Oops, I thought I looked through all comments before merging. But turns out I missed this one. @ziggie1984 can we add a unit test for this in a separate PR please?